src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,20 @@
 
 #include "t07_ip.h"
 #include <iostream>
-
+#include <string>
 
 using namespace std;
-
+string O(string z, int i, int d, string j) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'j' should be passed by const reference.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4685590)